pixel-point/pixelpoint-website
# feat: updated neon content

**This PR brings:**
- Neon case study content update

**Steps to test:**
- Visit [the page](https://deploy-preview-56--pixelpoint-website.netlify.app/case-studies/neon/)
- Ensure it has the right content and media assets
